src/modules/menu/components/share/MenuShareInputCopyButton.vue

```diff
@@ -1,52 +1,51 @@
 <template>
-    <div v-if="shortLink">
-        <label v-if="withText">{{ $t('share_link') }}: </label>
+    <div v-if="inputText">
+        <label v-if="labelText">{{ labelText }}: </label>
```

Comment:
shouldn't you be running `labelText` through `$t(...)` as it was before? Only the i18n key selection was made dynamic, but I don't see that `$t(...)` is called somewhere anymore

Reply:
The translation is done outside the component but you are right might be better to do it here for consistency

Reply:
Done